daemon/web/styles/index.sass

```diff
@@ -1,5 +1,6 @@
 @import ./colors
 @import ./typography
+<link href="https://fonts.googleapis.com/css?family=Rubik" rel="stylesheet">
```

Comment:
I believe this should be here 😋 

https://github.com/ubclaunchpad/inertia/blob/master/daemon/web/index.html

There are a few others there already, we might be able to remove them for the time being:

```html
<head>
  <meta charset="UTF-8">
  <link href='https://fonts.googleapis.com/css?family=Roboto:400,300,500' rel='stylesheet' type='text/css'>
  <link href="https://fonts.googleapis.com/css?family=Ubuntu+Mono" rel="stylesheet">
  <title>Inertia Web</title>
</head>
```

Reply:
@asitu make sure you remove this line! It is currently causing builds to fail:

```
@import ./typography
                      ^
      Invalid CSS after '..."./typography";': expected 1 selector or at-rule, was '<link href="https:/'
      in /go/src/github.com/ubclaunchpad/inertia/daemon/web/styles/index.sass (line 2, column 24)
 @ ./styles/index.sass 2:14-117
```

Reply:
Once builds are going again I think this is good to merge for the time being - we can work on the cleaning up the stylesheets this weekend